doc/sections/commands.md

```diff
@@ -257,7 +257,7 @@ set debug -s
 
 Each functionality is represented by a single character. For instance,
 `i` stands for type inference. To get the list of debuggable functionalities,
-do `lambdapi -h`.
+do `lambdapi -help`.
```

Comment:
-h is an option that works. -h and -help are added by ocaml but only -help is listed. I don't know if it is possible to change that. -help is not standard though. long option names should be prefixed by -- like --help. So, I propose to use --help instead.

Reply:
No, `-h` does not work. Like with every other unknown options, an error message is shown and the list of valid options is given. We do have `-help` and `--help` which are both added by default (see [here](https://caml.inria.fr/pub/docs/manual-ocaml/libref/Arg.html)), and that behavior is hard to change.

Reply:
Ah right. I didn't see the error message (shouldn't we put them in red?). So we have to use --help indeed.

Reply:
I don't think we easily can since the `Arg` module produces the error message. But yes, we can document `--help` instead.